apps/ledger-live-mobile/android/app/src/main/jni/MainComponentsRegistry.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)